app/controllers/catalog_controller.rb

```diff
@@ -135,16 +135,17 @@ def self.modified_field
     config.add_show_field 'contents_note_tesim'
     config.add_show_field 'dimensions_tesim'
     config.add_show_field 'director_tesim'
+    config.add_show_field 'edition_tesim', label: 'Edition'
```

Comment:
edition is multiple, not searchable and string so it should be _ssm which means string, stored,multiple
tesim means text,stored,indexed(searchable),multiple